weatherman.py

```diff
@@ -0,0 +1,61 @@
+import argparse
+from code_files.weather_parser import WeatherParser
+from code_files.weather_reporter import WeatherReporter
+from code_files.report_calculator import ReportCalculator
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser()
+    parser.add_argument("extract_to", type=str)
+    parser.add_argument("-e", "--extremes", type=int)
+    parser.add_argument("-a", "--averages", type=str)
```

Comment:
Let's make it more useful by allowing multiple dates to be passed.

Reply:
[Allowing multiple dates to be passed](https://github.com/ttqureshi/WeatherMan/pull/1/commits/6f2c4b5a1bdecc27fab75a9c7b9eb714a6baad70)